CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
  - add django rest framework to promote/revoke moderators for current LTIContext
  - add new role moderator and permission to manage moderators
 
+### Fixed
+ - fix sorting on sticky and announcements topics
```

Comment:
It is not because of this PR, but I see 2 `Added` sections in the Changelog.
Maybe it's an occasion to fix this?

Reply:
I agree with you, but I already fixed it on another current PR which is almost validated, the pr for the upload